app/controllers/authenticate_controller.rb

```diff
@@ -200,7 +200,10 @@ def handle_authentication_error(err)
 
   def log_backtrace(err)
     err.backtrace.each do |line|
-      logger.debug(line)
+      # We want to print a minimal stack trace in INFO level so that it is easier
+      # to understand the issue. We still want to print the full stack trace
+      # (including the ryb gems code) so we print it in DEBUG level
+      line.include?("2.5.0/gems") ? logger.debug(line) : logger.info(line)
```

Comment:
I don't think I have any security concerns here. However, are we setting ourselves up to break this when we update Ruby versions?  Can we dynamically get this string at runtime somehow?

Reply:
@micahlee @jonahx do you have an idea? i couldn't find a solution for this

Reply:
Hey @orenbm, it looks like the `GEM_HOME` environment variable would be a good way to provide this in a maintainable way.

That is currently `/var/lib/ruby/lib/ruby/gems/2.5.0`.

Reply:
thanks, fixed the code to use that.